src/mavedb/worker/jobs.py

```diff
@@ -247,6 +248,12 @@ async def create_variants_for_score_set(
 #  Mapping variants
 ####################################################################################################
 
+ANNOTATION_LAYERS = {
+    "g": "genomic",
+    "p": "protein",
+    "c": "cdna",
+}
```

Comment:
Could we move this to a new file `src/mavedb/lib/mapping.py`? Along with this layer dictionary, we can also move the VRSMap class from `src/mavedb/data_providers/services.py` to that file. That would better centralize these objects for use across the code base.